### ISSUE TITLE
Add an "upload" button for allowing users to upload a new file submission.

### ISSUE THREAD

**Issue by sheenarbw** [Umuzi-org/Tilde#287]
The button should just bring up a modal that displays a form where a person can add an upload. 

The button should only show up if the project card has the file upload submission type.

Dont worry about submitting the form just yet, that will be another issue

Blocked by #286
